CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# 2.1.0
+> Published 11 August 2022
+
+* Add YAML Configuration Format Support ([KTOR-3572](https://youtrack.jetbrains.com/issue/KTOR-3572))
```

Comment:
there is no change about reverting default behavior of string sent as json

Reply:
Because there is no issue for it

Reply:
https://github.com/ktorio/ktor/pull/3120/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR9